unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuildModeHUD/Scripts/BuildModeHUDController.cs

```diff
@@ -162,7 +161,7 @@ private void ConfigureTopActionsButtonsController()
         controllers.topActionsButtonsController.OnResetClick += () => OnResetAction?.Invoke();
         controllers.topActionsButtonsController.OnDuplicateClick += () => OnDuplicateSelectedAction?.Invoke();
         controllers.topActionsButtonsController.OnDeleteClick += () => OnDeleteSelectedAction?.Invoke();
-        controllers.topActionsButtonsController.OnLogOutClick += () => OnLogoutAction?.Invoke();
+        controllers.topActionsButtonsController.OnLogOutClick += ExitStart;
```

Comment:
We should unsubscribe from the event as well!

Reply:
In this case, in the same way as the rest of subscriptions from the different controllers, it is not needed because all these controllers are linked to the creation/destruction of the own BuidMode.

Anyway maybe we should review all the initial subscriptions done at the beggining of the HUD in order to check if we have some that we have missing its unsubscription (I'll note it for a separate PR).